.gitlab-ci.infra.yml

```
@@ -0,0 +1,34 @@
+stages:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)